src/models/forms.js

```diff
@@ -13,6 +13,10 @@ export const formByIdSchema = Joi.object()
   })
   .required()
 
+export const deleteFormPayloadSchema = Joi.object({
+  force: Joi.boolean().default(false)
+})
```

Comment:
@davidjamesstone your Joi-fu is better than mine. Is there a way I can make Joi allow a completely empty payload, rather than requiring `{}` to be supplied? I could tack on `.allow(null)` to the end but that makes the entire payload optional. I want Joi to assign `force: false` even if the payload is completely missing, I couldn't find a nice way to do that though.

Reply:
```
const schema = Joi.alternatives().conditional(Joi.any().valid(null), {
    then: Joi.object().default({ force: false }),
    otherwise: Joi.object({
      force: Joi.boolean().default(false)
    }).required()
})
```

How about something like this, using `joi.alternatives`?

Reply:
Thanks for the help Dave! Fixed.